modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -36,13 +36,13 @@
 `ARGOCD_AUTH_TOKEN2`
 
 
-|GitHub  |`https://npmjs.com/package/@backstage/plugin-catalog-backend-module-github/v/0.7.8[@backstage/plugin-catalog-backend-module-github]` |0.7.8 
+|GitHub  |`https://npmjs.com/package/@backstage/plugin-catalog-backend-module-github/v/0.7.9[@backstage/plugin-catalog-backend-module-github]` |0.7.9 
```

Comment:
It looks like we picked up a plugin version that is from a new backstage version that 1.4.2 supports.  
[v0.7.6 corresponds to BS v1.32.6](https://github.com/backstage/backstage/blob/v1.32.6/plugins/catalog-backend-module-github/package.json#L3) which is what [we shipped in 1.4.0](https://github.com/redhat-developer/rhdh/blob/72d5be99386498892a38ee8f64879f28fc842788/dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-github-dynamic/package.json#L3)
[v0.7.9 corresponds to BS v1.35.1](https://github.com/backstage/backstage/blob/v1.35.1/plugins/catalog-backend-module-github/package.json#L3)